docs/source/redhat-opens-install.rst

```diff
@@ -111,6 +111,9 @@ please get in touch with us and we'll be happy to help you through the process.
 
    - Download, unpack, and install the binary::
 
+.. warning:: We've seen certificate errors downloading etcd - you may need to
+             add --insecure to the `curl` command to ignore this.
+
```

Comment:
@ed-harrison I’ve made a commit which tries to fix the indentation, by moving the warning to after the curl command. I think it’s worth having the warning, because just dropping the `--insecure` flag without explanation is bound to raise some eyebrows.

This is what the rendered output looks like on my machine; could you double-check the change please?

![screen shot 2015-06-18 at 20 09 43](https://cloud.githubusercontent.com/assets/301220/8239782/fb83e1e8-15f5-11e5-8011-7626cce95e13.png)